lib/msal-core/README.md

```diff
@@ -18,14 +18,31 @@ Via NPM:
 Via CDN:
 
     <!-- Latest compiled and minified JavaScript -->
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.js"></script>
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
+    <script type="text/javascript" src="https://{CDN-URL}/msal.js" integrity="sha{alg}-{checksum}" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://{CDN-URL}/msal.min.js" integrity="sha{alg}-{checksum}" crossorigin="anonymous"></script>
+
+Latest Microsoft CDN Version:
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.1.3/js/msal.js" integrity="sha384-3cT22wEPS/umVHpe5hGwbRF8PLObcSXaaFncKfbzutTBzn/aEMQ6aIlxaa6J2TG1" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.1.3/js/msal.min.js" integrity="sha384-kYijiCrNWywvKX1VI7259ktIf0ebXhlDeVD2dBEX+GeVbmY1GEboq3dsDDvYuDP/" crossorigin="anonymous"></script>
+
+Note: We recommend including the [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) Hash with all script tags when using `msal.js` or `msal.min.js`. When providing the SRI Hash, you *must* also provide the `crossorigin="anonymous"` field in the same tag.
```

Comment:
I think the note about resilience below should just be included here, instead of splitting it up in two sections of the readme.

Also, I think you should very clear note three things:
1. SRI hashes are optional.
2. Each hash is unique to the version of `msal`.
3. A link to the wiki page with the hashes for old version.

Reply:
Added this to the README. The wiki page hasn't been created yet, so I will add the link as soon as it is available.